en/mkdocs.yml

```diff
@@ -39,7 +39,7 @@ strict: false
 nav:
   - Home: index.md
   - Get started:
-      - Install the Ballerina extension: get-started/install.md
+      - Install: get-started/install.md
```

Comment:
Shouldn't we match the page titles here? Page title is `Install the Ballerina extension`?